app/assets/stylesheets/recipe_show.scss

```diff
@@ -0,0 +1,70 @@
+@import "colors";
+
+.recipe-info-block {
+  padding-bottom: 20px;
+}
+
+.recipe-info-attr {
+  padding: 0px 5px;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, padding<br>`0px` should be written without units as `0`